processor/src/main/java/org/mapstruct/ap/internal/conversion/BigDecimalToStringConversion.java

```diff
@@ -19,15 +19,16 @@
 package org.mapstruct.ap.internal.conversion;
 
```

Comment:
`IMPORT` does the trick for a couple of other spots 👍. Unfortunately, it doesn't seem to treat the `import static` properly and this part slips by... 😁